src/Public/OpenApi.ps1

```diff
@@ -1304,77 +1967,203 @@ The Property that need converting (such as from New-PodeOAIntProperty).
 .PARAMETER Reference
 The name of an existing component parameter to be reused.
 
+.PARAMETER ContentSchemas
+The content-types and the name of an existing component schema to be reused.
+
+.PARAMETER Explode
+If supplied, controls how arrays are serialized in query parameters
+
 .EXAMPLE
 New-PodeOAIntProperty -Name 'userId' | ConvertTo-PodeOAParameter -In Query
 
 .EXAMPLE
 ConvertTo-PodeOAParameter -Reference 'UserIdParam'
+
+.EXAMPLE
+ConvertTo-PodeOAParameter  -In Header -ContentSchemas @{ 'application/json' = 'UserIdSchema' }
+
 #>
 function ConvertTo-PodeOAParameter
 {
-    [CmdletBinding(DefaultParameterSetName='Reference')]
-    param(
-        [Parameter(Mandatory=$true, ParameterSetName='Schema')]
+    [CmdletBinding(DefaultParameterSetName = 'Reference')]
+    param( 
+        [Parameter(Mandatory = $true, ParameterSetName = 'Schema')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'ContentSchemas')]
         [ValidateSet('Cookie', 'Header', 'Path', 'Query')]
         [string]
         $In,
 
-        [Parameter(Mandatory=$true, ValueFromPipeline=$true, ParameterSetName='Schema')]
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ParameterSetName = 'Schema')]
         [ValidateNotNull()]
         [hashtable]
         $Property,
 
-        [Parameter(Mandatory=$true, ParameterSetName='Reference')]
+        [Parameter(Mandatory = $true, ParameterSetName = 'Reference')]
         [string]
-        $Reference
-    )
+        $Reference,
 
-    # return a reference
-    if ($PSCmdlet.ParameterSetName -ieq 'reference') {
-        if (!(Test-PodeOAComponentParameter -Name $Reference)) {
+        [Parameter(Mandatory = $true, ParameterSetName = 'ContentSchemas')]
+        [hashtable]
+        $ContentSchemas,
+
+        [Parameter() ]
+        [Switch]
+        $Explode, 
+
+        [Parameter() ]
+        [ValidateSet('simple', 'label', 'matrix', 'query', 'form', 'spaceDelimited', 'pipeDelimited', 'deepObject' )]
+        [string]
+        $Style 
```

Comment:
Just to match the style of other options, alter the casing of the first characters. Ie: Simple, SpaceDelimited, etc.